blkn/src/SystemMessage/IconWrapper.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  type: "warning" | "error" | "success" | "info",
+  icon: Function,
```

Comment:
type too vague, be specific

Reply:
Do you have any suggestions? Label? Outcome? Response?

Reply:
Ok shame for me you reffered to icon not type. sorry